### PR TITLE
amm: Cleanup some changes after merge

### PR DESCRIPTION
**Pull request by axic**
Some things missed in #591.

### CODE REVIEW

**Review by mijovic — COMMENTED** [review]
This looks good. Why it is a draft?

**Comment by axic** [comment]
Want to do a once-over of #591 if I catch anything else, to avoid creating dozens of small fix PRs.

File: contracts/amm/ITempusAMM.sol [hunk 1]
```diff
@@ -110,6 +109,8 @@ interface ITempusAMM is IERC20, IRateProvider, IOwnable {
     /// @return token1Balance Amount of Token1 corresponding to the LP tokens
     function compositionBalanceOf(address account) external view returns (uint256 token0Balance, uint256 token1Balance);
 
+    function getRate() external view returns (uint256);
```
Comment (axic) [inline contracts/amm/ITempusAMM.sol:112]: Is this function actually needed? And what would be a good natspec.

Comment (mijovic) [reply contracts/amm/ITempusAMM.sol:112]: Let's maybe remove this method completely?

Comment (axic) [reply contracts/amm/ITempusAMM.sol:112]: I can remove it in a few hours.

Comment (mijovic) [reply contracts/amm/ITempusAMM.sol:112]: Perfect

Comment (axic) [reply contracts/amm/ITempusAMM.sol]: Done.

**Review by mijovic — APPROVED** [review]

**Pull request merged**